src/main/kotlin/no/nav/dagpenger/soknad/orkestrator/søknad/SøknadService.kt

```diff
@@ -60,7 +61,15 @@ class SøknadService(
         sikkerlogg.info { "Publiserte melding om ny søknad med søknadId: $søknadId og ident: $ident" }
     }
 
-    internal fun slett(
+    internal fun slettSøknad(
```

Comment:
Burde denne hete `slettOgPubliser` eller noe sånt? Det høres ut som de to metodene gjør det samme,  men at den ene øker metrikken. Men sånn de brukes nå, er jo forskjellene egentlig om man skal publisere melding eller ikke, og begge metodene ender opp med å øke metrikken.